src/modules-lua/noit/module/ssh2.lua

```diff
@@ -48,23 +48,23 @@ function onload(image)
     <parameter name="method_kex"
                required="optional"
                default="diffie-hellman-group14-sha1"
-               allowed="^diffie-hellman-(?:group1-sha1|group14-sha1|group-exchange-sha1)$">The key exchange method to use.</parameter>
+               allowed="^diffie-hellman-(?:group1-sha1|group14-sha1|group16-sha512|group18-sha512)$">The key exchange method to use.</parameter>
     <parameter name="method_hostkey"
                required="optional"
                default="ssh-rsa"
-               allowed="^(?:ssh-dss|ssh-rsa)$">The host key algorithm supported.</parameter>
+               allowed="^(?:ssh-rsa|ecdsa-sha2-nistp256|ssh-ed25519|ssh-dss)$">The host key algorithm supported.</parameter>
     <parameter name="method_crypt_cs"
                required="optional"
-               allowed="^(?:aes256-ctr|aes192-ctr|aes128-ctr|aes256-cbc|aes192-cbc|aes128-cbc|blowfish-cbc|arcfour128|arcfour|cast128-cbc|3des-cbc|none)$">The encryption algorithm used from client to server.</parameter>
```

Comment:
why are these removed?

Reply:
I was cleaning up the list of rarely-used options.  Note that I did preserve all the aes*-{ctr,cbc}.